Doc/library/resource.rst

```diff
@@ -255,6 +255,14 @@ platform.
 
    .. versionadded:: 3.4
 
+.. data:: RLIMIT_KQUEUES
+
+   The maximum number of kqueues is allowed to create by this user id.
```

Comment:
it's still different than the sentence from the FreeBSD manual page, this sentence sounds wrong to me.